src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs

```diff
@@ -574,9 +574,10 @@ public void AddSourceFile(string fullPath, SourceCodeKind sourceCodeKind = Sourc
             string fullPath,
             SourceCodeKind sourceCodeKind = SourceCodeKind.Regular,
             ImmutableArray<string> folders = default,
+            bool designTimeOnly = default,
```

Comment:
```suggestion
            bool designTimeOnly = false,
```

A bit confusing to see 'default' when we know what that is.